resources/assets/lib/chat/create-announcement.tsx

```diff
@@ -88,12 +103,19 @@ export default class CreateAnnouncement extends React.Component<Props> {
               <BusySpinner busy={this.model.lookingUpUsers} />
             </div>
           </InputContainer>
-          <InputContainer model={this.model} modifiers={['chat', 'fill']} name='message'>
+          <InputContainer
+            for='chat-form-message'
+            labelKey='chat.form.labels.message'
+            maxLength={maxLengths.message}
+            model={this.model}
+            modifiers={['chat', 'fill']}
+            name='message'
+          >
             <textarea
               autoComplete='off'
               className='chat-form__input chat-form__input--box'
               defaultValue={this.model.inputs.message}
-              maxLength={maxLength}
+              id='chat-form-message'
```

Comment:
`id` (and input container's `for`) not needed here? It was in `chat-form-users` just because it also contains buttons